inc/html.php

```diff
@@ -104,9 +112,10 @@ function html_secedit($text, $show = true) {
  * @return string
  * @triggers HTML_SECEDIT_BUTTON
  */
-function html_secedit_button($matches){
+function html_secedit_button($matches)
+{
     $json = htmlspecialchars_decode($matches[1], ENT_QUOTES);
-    $data = json_decode($json, true);
+    $data = json_decode($json, true, 512, JSON_THROW_ON_ERROR);
```

Comment:
Logic change! Needs review.

Reply:
Ah this one was problematic. I add a try/catch block.